src/lib/github/gh-utility.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * function that returns basic repository properties from payload.
+ */
+
+import { Context } from "./types";
+
+export default function getBasicRepoProps (context: Context): any {
```

Comment:
Please address replacing the return type of `any` with a proper concrete type. Is my request clear ?